javascript/node/selenium-webdriver/example/google_search.js

```diff
@@ -47,4 +47,4 @@ driver.get('http://www.google.com/ncr');
 driver.findElement(By.name('q')).sendKeys('webdriver');
 driver.findElement(By.name('btnG')).click();
 driver.wait(until.titleIs('webdriver - Google Search'), 1000);
-driver.quit();
```

Comment:
Actually adding a CR after the last character, and not sure why GitHub's
web editor chose to do that. Feel free to merge that out or I can squash if
you'd like.
On Mar 12, 2016 7:39 AM, "Dj" <notifications@github.com> wrote:

> In javascript/node/selenium-webdriver/example/google_search.js
> <https://github.com/SeleniumHQ/selenium/pull/1789#discussion_r55922204>:
>
> > @@ -47,4 +47,4 @@ driver.get('http://www.google.com/ncr');
> >  driver.findElement(By.name('q')).sendKeys('webdriver');
> >  driver.findElement(By.name('btnG')).click();
> >  driver.wait(until.titleIs('webdriver - Google Search'), 1000);
> > -driver.quit();
>
> why are we removing the last line at EOF?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SeleniumHQ/selenium/pull/1789/files#r55922204>.
>